cypress/support/step-definitions/checkbox-steps.js

```diff
@@ -141,7 +143,51 @@ When('I check {string} checkbox {int} times', (position, times) => {
   }
 });
 
-Then('checkbox on preview is {string}', (text) => {
+When('I hover mouse onto {string} icon in no iFrame for checkbox', (position) => {
+  switch (position) {
+    case 'error':
+      checkboxByID('required').trigger('mouseover');
+      break;
+    case 'warning':
```

Comment:
to avoid duplication we could probably change it to 
```
case 'warning', 'info', 'optional':
       checkboxByID(position).trigger('mouseover');
       break;

Reply:
fixed